evdev_controller:
  ros__parameters:
    hz: 50.0   # Rate

    event: discovery_event   # Evdev path
    controller_name: slide_winder # Name of controller, if discovery ignored

    publishers:
      button_subset: /teleop_cmd/button/
      axis_subset: /teleop_cmd/axis/

    services:
      is_init: /evdev_controller/is_init
      stop: /evdev_controller/stop